pkg/harvester/edit/harvesterhci.io.setting.vue

```diff
@@ -296,4 +296,9 @@ export default {
 :deep() .edit-help code {
   padding: 1px 5px;
 }
+
+::v-deep(.banner__content.closable) {
+  white-space: normal;
+  word-break: break-word;
+}
```

Comment:
To fix the overflow issue in error banner.
<img width="1920" height="968" alt="Screenshot 2025-10-30 at 1 33 52 PM (2)" src="https://github.com/user-attachments/assets/3172c0e1-ec06-4be3-b21c-93d2437b5f90" />